### DISCUSSION TITLE
Questão TI21#12

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#19] [Q&A] [answered]
Qual é a diferença entre uma variável let, const e var em JavaScript?

**Comment by RenonHG** [comment]
Variáveis declaradas com var podem ser atualizadas, enquanto variáveis declaradas com const não podem, e let é como se fosse temporária, ela só funcionará para aquele escopo uma vez e depois será "descartada" por assim dizer.

**Comment by erickgusson** [comment, marked as answer]
Let é uma variavel temporaria e contextual, se você criar ela dentro de um if, você não consegue usar ela fora
Var é uma variavel global, se você definir ela dentro de um if da para usar ela fora
Const é uma constante que nunca muda, ela é ideal se o valor sempre for o mesmo
